### ISSUE TITLE
cannot move image tab

### ISSUE THREAD

**Issue by mark-hahn** [atom/image-view#36]
When I load an image (using the file picker) and then try to drag the image tab over another tab I get errors.  The first time it didn't move and an untitled tab was created between the two tabs.  The second time atom crashed with the exception below.

ubuntu 14.0.04
atom 205
safe mode
apm  0.168.0
npm  2.5.1
node 0.10.35
python 2.7.3
git 1.7.9.5

```
App load time: 127ms
ATTENTION: default value of option force_s3tc_enable overridden by environment.
[11890:0604/130048:ERROR:sandbox_linux.cc(325)] InitializeSandbox() called with multiple threads in process gpu-process
[11907:0604/130048:INFO:renderer_main.cc(212)] Renderer process started
[11910:0604/130048:INFO:renderer_main.cc(212)] Renderer process started
[11913:0604/130048:INFO:renderer_main.cc(212)] Renderer process started
[11714:0604/130050:INFO:CONSOLE(52)] "Window load time: 1163ms", source: file:///usr/share/atom/resources/app.asar/static/index.js (52)
[11714:0604/130050:INFO:CONSOLE(52)] "Window load time: 1257ms", source: file:///usr/share/atom/resources/app.asar/static/index.js (52)
[11714:0604/130050:INFO:CONSOLE(52)] "Window load time: 1265ms", source: file:///usr/share/atom/resources/app.asar/static/index.js (52)
[11714:0604/130119:ERROR:browser_main_loop.cc(212)] Attempting to store changes into `/root/.local/share/recently-used.xbel', but failed: Failed to create file '/root/.local/share/recently-used.xbel.1JNXZX': No such file or directory (http://bugs.chromium.org/161366)
[11714:0604/130119:ERROR:browser_main_loop.cc(212)] Attempting to set the permissions of `/root/.local/share/recently-used.xbel', but failed: No such file or directory (http://bugs.chromium.org/161366)
[11714:0604/130119:ERROR:browser_main_loop.cc(212)] Attempting to store changes into `/root/.local/share/recently-used.xbel', but failed: Failed to create file '/root/.local/share/recently-used.xbel.DFNXZX': No such file or directory (http://bugs.chromium.org/161366)
[11714:0604/130119:ERROR:browser_main_loop.cc(212)] Attempting to set the permissions of `/root/.local/share/recently-used.xbel', but failed: No such file or directory (http://bugs.chromium.org/161366)
```

**Comment by thomasjo** [comment]
Are you able to reproduce this in safe mode?

**Comment by mark-hahn** [comment]
Yes.  I mentioned that in my version list, which is probably not where it
belonged.

**Comment by thomasjo** [comment]
> Yes.  I mentioned that in my version list

Sorry, I missed completely missed that.

**Comment by thomasjo** [comment]
I'm unable to reproduce this on OS X 10.10.3 running Atom v0.206.0-65831f1.
Does this happen with multiple images and/or formats? For reference I opened 4 huge JPEG images, and moved them around without any issues.

**Comment by mark-hahn** [comment]
It has happened every time for me today.  I haven't used images in some
time so I don't know when it started.

I will try to narrow down the situation.  This is a bit high-priority for
me since one of my packages relies on dragging an image tab into a markdown
tab.

**Comment by mark-hahn** [comment]
It's worse than I thought.  There doesn't have to be any other tab open.
If I have one image tab I can click on it, and the tooltip shows on
hovering, but dragging crashes atom instantly.  I assume it is triggered by
the mousemove event.

I set the dev inspector to pause on any exception, including caught ones,
but it didn't change anything and didn't break.

I tried a lot of dragging with non-image tabs and couldn't see any problem.

I will install this repo and walk through the code with the debugger.

**Comment by mark-hahn** [comment]
I may not be up to the task to chase this down.  I can't even get this package to run in dev mode. 

When I installed this package using `apm develop image-view` things broke more.  When I open a .jpg file with the file-picker atom treats it as a text file and spits out garbage.  Assuming there was some problem with the opener, I tried to debug the activation.  But I can't get the main file to show in the dev inspector.  I put in a `debugger` statement but it isn't hit.

This makes me think that atom isn't really using the code from `~/.atom/dev/packages/image-view`.   Is there some way to verify this is the case?    I am opening atom with `--dev`. 

Also, is the version I'm getting with `apm develop image-view` the same as the one released in atom 205?  It is version 0.54.0 in the `package.json` file in `~/.atom/dev/packages/image-view`.
